open-banking-accelerator/accelerators/ob-apim/README.md

```diff
@@ -0,0 +1,17 @@
+### **WSO2 Open Banking APIM Accelerator 3.0.0**
+
+**Try Locally:**
+
+Prerequisites:
+1. Download WSO2 API Manager product 
+2. Download the WSO2 Open Banking API Management Accelerator 
+3. Setup MySQL database server
+4. Install Java on your local machine
+
+Steps:
+1. Extract WSO2 APIM <WSO2_APIM_HOME>
```

Comment:
Shall we add a step after this to copy the APIM Accelerator packs to the APIM bin folder